runtime/kusama/constants/src/lib.rs

```diff
@@ -26,8 +26,8 @@ pub mod currency {
 	pub const EXISTENTIAL_DEPOSIT: Balance = 1 * CENTS;
 
 	pub const UNITS: Balance = 1_000_000_000_000;
-	pub const CENTS: Balance = UNITS / 30_000;
-	pub const QUID: Balance = CENTS * 100;
+	pub const QUID: Balance = UNITS / 30;
+	pub const CENTS: Balance = QUID / 100;
```

Comment:
This change makes the values more intuitive to me, but does change all the derivative values by 10x, and they are used in other places in the runtime (ED, crowdloan deposit, etc.). I can make a PR to this branch to update those such that they remain unchanged.

Reply:
Will need Cumulus companion too as Statemine pulls these in.

Reply:
IMO updating the deposits on the tracks is better.
https://github.com/paritytech/polkadot/pull/6386

Reply:
These values are meant to loosely track the price, so no, changing these is, and always was, The Plan.